[2022-02-24 23:47:26,872] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [queued]>
[2022-02-24 23:47:26,883] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [queued]>
[2022-02-24 23:47:26,883] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 23:47:26,883] {taskinstance.py:1244} INFO - Starting attempt 6 of 7
[2022-02-24 23:47:26,884] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 23:47:26,895] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): create_fuel_dataset> on 2022-02-24 22:55:20.271881+00:00
[2022-02-24 23:47:26,902] {standard_task_runner.py:52} INFO - Started process 2607 to run task
[2022-02-24 23:47:26,909] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'daily_fuel_data_ingestion', 'create_fuel_dataset', 'manual__2022-02-24T22:55:20.271881+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/ingest_daily_fuel_data_dag.py', '--cfg-path', '/tmp/tmp2mp7mib0', '--error-file', '/tmp/tmp9580nr7s']
[2022-02-24 23:47:26,910] {standard_task_runner.py:77} INFO - Job 31: Subtask create_fuel_dataset
[2022-02-24 23:47:26,969] {logging_mixin.py:109} INFO - Running <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [running]> on host 036a2a858d52
[2022-02-24 23:47:27,038] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=daily_fuel_data_ingestion
AIRFLOW_CTX_TASK_ID=create_fuel_dataset
AIRFLOW_CTX_EXECUTION_DATE=2022-02-24T22:55:20.271881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-24T22:55:20.271881+00:00
[2022-02-24 23:47:28,138] {warnings.py:110} WARNING - /opt/***/dags/ingest_daily_fuel_data_dag.py:77: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  final_df["national_average"] = final_df.mean(axis=1).round(1)

[2022-02-24 23:47:28,145] {logging_mixin.py:109} INFO - CSV saved to /opt/***/daily_fuel_data_{{ execution_date.strftime('%d-%b-%Y') }}.csv
[2022-02-24 23:47:28,145] {python.py:175} INFO - Done. Returned value was: None
[2022-02-24 23:47:28,159] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=daily_fuel_data_ingestion, task_id=create_fuel_dataset, execution_date=20220224T225520, start_date=20220224T234726, end_date=20220224T234728
[2022-02-24 23:47:28,204] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-24 23:47:28,246] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
